Library-SMTLIB/src/de/uni_freiburg/informatik/ultimate/logic/Theory.java

```diff
@@ -856,6 +856,32 @@ public Sort getResultSort(final String[] indices, final Sort[] paramSorts, final
 				return paramSorts[0];
 			}
 		}
+		class Bv2IntFunction extends FunctionSymbolFactory {
+			public Bv2IntFunction(final String name) {
+				super(name);
+				assert name.equals("bv2int") : "Wrong name: " + name;
+			}
+
+			@Override
+			public Sort getResultSort(final String[] indices, final Sort[] paramSorts, final Sort resultSort) {
+				return mNumericSort;
```

Comment:
There should be type-checking rules here:

```suggestion
				if (indices != null || paramSorts.length != 1 || !paramSorts[0].getName() != "BitVec" || resultSort != null) {
					return null;
				}
				return mNumericSort;				
```